Content.Server/Explosion/Components/ShockOnTriggerComponent.cs

```diff
@@ -0,0 +1,36 @@
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom;
+
+namespace Content.Server.Explosion.Components;
+
+[RegisterComponent, AutoGenerateComponentPause]
+public sealed partial class ShockOnTriggerComponent : Component
+{
+    /// <summary>
+    /// The force of an electric shock when the trigger is triggered.
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField("force")]
```

Comment:
```suggestion
    [DataField]
```

dont add datafield names